src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -115,6 +115,13 @@
       + "is automatically destroyed.";
   public static final String CONSUMER_INSTANCE_TIMEOUT_MS_DEFAULT = "300000";
 
+  public static final String SIMPLE_CONSUMER_POOL_SIZE_CONFIG = "simpleconsumer.pool.size";
+  private static final String
+      SIMPLE_CONSUMER_POOL_SIZE_DOC =
+      "maximum number of SimpleConsumers that can be instantiated by broker."
+      + " If empty, then no pool is used.";
+  public static final String SIMPLE_CONSUMER_POOL_SIZE_DEFAULT = "3";
```

Comment:
It depends a lot on how the SimpleConsumer API will be used. In my case this API would be used only for debugging purposes, and therefore I wouldn't require a big pool size. What value would you use ?